utils/AsyncIterableTask.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable functional/prefer-readonly-type */
+/* eslint-disable @typescript-eslint/array-type */
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-let */
+/* eslint-disable prefer-const */
+// -------------------------------------------------------------------------------------
+// model
+// -------------------------------------------------------------------------------------
+
+import * as T from "fp-ts/lib/Task";
+import * as TE from "fp-ts/lib/TaskEither";
+
+import { mapAsyncIterable } from "@pagopa/io-functions-commons/dist/src/utils/async";
+import { pipe } from "fp-ts/lib/function";
+
+/**
+ * @category model
+ * @since 2.0.0
+ */
+export type AsyncIterableTask<A> = T.Task<AsyncIterable<A>>;
+
+export const fromAsyncIterable = <A>(
+  a: AsyncIterable<A>
+): AsyncIterableTask<A> => T.of(a);
+
+/**
+ * `map` can be used to turn functions `(a: A) => B` into functions `(fa: F<A>) => F<B>` whose argument and return types
+ * use the type constructor `F` to represent some computational context.
+ *
+ * @category Functor
+ * @since 2.0.0
```

Comment:
I think `since 2.0.0` it's referred to fp-ts 2.0, so you can remove it.
```suggestion
```

Reply:
Done in [280f52a](https://github.com/pagopa/io-functions-services-cache/pull/32/commits/280f52a5d30b7dd67e5ac9556e1d271d9989e261)